### PR TITLE
Restructure build output to single folder

### PR DESCRIPTION
**Pull request by jthomperoo**

### CODE REVIEW

**Review by mbengit — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -5,14 +5,15 @@ QJSLib is a JavaScript utility library that provides functionality for QRadar ap
...
 * [CHANGELOG](CHANGELOG.md)
+* [DOCUMENTATION](https://github.com/IBM/qjslib/wiki/qappfw)
 
 ## How to use
 ### Browser vs module
-The build process for this project generates two seperate *qappfw.js* files, one in *build/* and one in *lib/*. 
+The build process for this project generates two seperate files in *lib/*.
 
-The file in *lib/* is not minified and designed to be bundled with your application code.
+The file in *qappfw.js* is not minified and designed to be bundled with your application code.
```
Comment (wilhaddenIBM) [inline README.md]: Bit of a weird sentence

File: README.md [hunk 2]
```diff
@@ -5,14 +5,15 @@ QJSLib is a JavaScript utility library that provides functionality for QRadar ap
...
 
-The file in *build/* contains a number of polyfills and is minified to ensure compatability with older browsers, and as such is suitable for direct use from the browser. 
+The file in *qappfw.min.js* contains a number of polyfills and is minified to ensure compatability with older browsers, and as such is suitable for direct use from the browser.
```
Comment (wilhaddenIBM) [inline README.md]: Also weird

**Review by wilhaddenIBM — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jthomperoo) [reply README.md]: *qappfw.js* is not minified, it is designed to be bundled with your application code.

Does this read a bit better?

File: README.md (hunk 2 above)
Comment (jthomperoo) [reply README.md]: *qappfw.min.js* is minified with polyfills for browser compatibility, it is designed to be used directly in the browser.

How about this?

File: README.md (hunk 1 above)
Comment (mbengit) [reply README.md]: qappfw.js is not minified. It is designed to be bundled with your application code.

File: README.md (hunk 2 above)
Comment (mbengit) [reply README.md]: ...compatibility. It is designed....

File: README.md [hunk 3]
```diff
@@ -5,14 +5,15 @@ QJSLib is a JavaScript utility library that provides functionality for QRadar ap
 * [CONTRIBUTING](CONTRIBUTING.md)
 * [MAINTAINERS](MAINTAINERS.md)
 * [CHANGELOG](CHANGELOG.md)
+* [DOCUMENTATION](https://github.com/IBM/qjslib/wiki/qappfw)
 
 ## How to use
 ### Browser vs module
-The build process for this project generates two seperate *qappfw.js* files, one in *build/* and one in *lib/*. 
+The build process for this project generates two seperate files in *lib/*.
```
Comment (mbengit) [inline README.md]: Mis-spelling: separate

**Review by mbengit — APPROVED** [review]

**Review by wilhaddenIBM — APPROVED** [review]

**Pull request merged**